components/input-group/docs.mdx

```diff
@@ -0,0 +1,92 @@
+---
+name: InputGroup
+menu: Components
+route: /input-group
+---
+
+import { Playground, PropsTable } from 'docz';
+import { InputGroup, InputGroupAddon } from './src';
+import { Button } from '../button/src';
+import { TextInput } from '../text-input/src';
+
+# InputGroup
+
+Styled input fields with attached addons. Addons can be text (\$, %, .00 etc) or form controls (buttons or select inputs).
```

Comment:
Remove `\$` slash

Reply:
<img width="844" alt="Screen Shot 2019-09-11 at 4 01 29 pm" src="https://user-images.githubusercontent.com/2049709/64672209-21eba480-d4ae-11e9-8e86-d88b8fda1792.png">

Reply:
I cannot remove the / 🤔
Prettier put it back...